x-pack/plugins/apm/server/routes/services.ts

```diff
@@ -470,3 +473,82 @@ export const serviceDependenciesRoute = createRoute({
     });
   },
 });
+
+export const serviceProfilingTimelineRoute = createRoute({
+  endpoint: 'GET /api/apm/services/{serviceName}/profiling/timeline',
+  params: t.type({
+    path: t.type({
+      serviceName: t.string,
+    }),
+    query: t.intersection([
+      rangeRt,
+      uiFiltersRt,
+      t.partial({
```

Comment:
There's an `environmentRt` that does the same thing.

Reply:
nice 👍